src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OCSP.cs

```diff
@@ -44,7 +44,7 @@ internal static X509VerifyStatusCode X509ChainGetCachedOcspStatus(SafeX509StoreC
         {
             X509VerifyStatusCode response = CryptoNative_X509ChainGetCachedOcspStatus(ctx, cachePath, chainDepth);
 
-            if (response < 0)
+            if (response.Code < 0)
             {
                 Debug.Fail($"Unexpected response from X509ChainGetCachedOcspSuccess: {response}");
```

Comment:
Should this debug now show `{response.Code}`? I tried this printing in a hello-world-style program and `{response}` seems to get rendered as the class name: `Interop+Crypto+X509VerifyStatusCode`. Maybe a `ToString` method will help?

Reply:
Ah, yeah, a ToString() (probably just using the X509VerifyStatusCodeUniversal ToString) would probably help.

Just checking: Did you actually hit that Debug.Fail, or just a similar kind of problem?

Reply:
I was just skimming this code trying to learn :smile: